Source/Core/Common/QoSSession.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/QoSSession.h"
+#include "Core/ConfigManager.h"
+
+#if defined(_WIN32)
+#include <Qos2.h>
+#pragma comment(lib, "qwave")
```

Comment:
i think you need to make sure this is in the minimal qt redist which dolphin uses

Reply:
It should be included with Windows >= Vista